db/migrate/20160311174929_generate_sequences_for_option_nodes.rb

```diff
@@ -0,0 +1,15 @@
+class GenerateSequencesForOptionNodes < ActiveRecord::Migration
+  def up
+    OptionSet.find_each do |option_set|
+      option_set.generate_sequence(batch: true)
```

Comment:
This is fine for a migration I think. So I guess we're keeping batch mode, but can you please change generate_sequence to use find_each somehow? Loading 200k options into memory at once will likely bring down the server.